corehq/apps/userreports/pillow.py

```diff
@@ -70,19 +70,29 @@ def rebuild_tables_if_necessary(self):
                 migration_context = get_migration_context(connection, table_map.keys())
                 diffs = compare_metadata(migration_context, metadata)
 
-            tables_to_rebuild = get_tables_to_rebuild(diffs, table_map.keys())
+            tables_to_rebuild = set(get_tables_to_rebuild(diffs, table_map.keys()))
```

Comment:
i didn't dig in to that too closely. @snopoke understands how that works, but basically I think `compare_metadata` returns multiple diffs for the same table and then they weren't being reconciled. I suspect something could be changed around [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/fluff/signals.py#L75) or we could just add the `set` call [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/fluff/signals.py#L79)